tests/1_login.spec.ts

```diff
@@ -1,13 +1,9 @@
-import { expect, test } from "@playwright/test";
-import { LoginPage } from "../pages/loginPage";
+import { test } from "../fixtures";
+import { expect } from "@playwright/test";
 import { config } from "../env.config";
 
-test("Verify login with valid credentials", async ({ page }) => {
-  const loginPage = new LoginPage(page);
-
-  await page.goto(`${config.weburl}/auth/login`);
-  await loginPage.login(`${config.useremail}`, `${config.userpassword}`);
-  await expect(page).toHaveURL("/account");
+test("Verify login with valid credentials", async ({ loggedInPage, loginPage }) => {
```

Comment:
```suggestion
test("Verify login with valid credentials", async ({ loggedInPage, loginPage }) => {
```

loggedInPage and loginPage - sounds weird. Lets rename loggedInPage fixture to loggedApp, for example